app/models/manageiq/providers/ansible_tower/inventory/collectors/automation_manager.rb

```diff
@@ -0,0 +1,19 @@
+class ManageIQ::Providers::AnsibleTower::Inventory::Collectors::AutomationManager < ManagerRefresh::Inventory::Collector
+  def inventories
+    manager.with_provider_connection do |connection|
+      connection.api.inventories.all
+    end
+  end
+
+  def hosts
+    manager.with_provider_connection do |connection|
+      connection.api.hosts.all
+    end
+  end
+
+  def job_templates
+    manager.with_provider_connection do |connection|
```

Comment:
If we know we're going to call all of these methods in succession (e.g.: [here](https://github.com/ManageIQ/manageiq/pull/13728/files#diff-fcc06c725d5c5df7f17b2c164f36d607R2)) could we keep the connection open so we don't have to open/close it every time?  I'm thinking for mainly about rhev where it has to connect to a remote system without a connection broker the cost to open/close the sessions can add up.